helper-cli/src/main/kotlin/commands/ListCopyrightsCommand.kt

```diff
@@ -55,12 +57,31 @@ internal class ListCopyrightsCommand : CommandWithHelp() {
     )
     private var copyrightGarbageFile: File? = null
 
+    @Parameter(
+        names = ["--package-id"],
+        order = PARAMETER_ORDER_OPTIONAL,
+        converter = IdentifierConverter::class,
+        description = "The target package for which the copyrights shall be listed."
+    )
+    private var packageId: Identifier? = null
+
+    @Parameter(
+        names = ["--license-id"],
+        order = PARAMETER_ORDER_OPTIONAL,
+        description = "The license for which the copyrights shall be listed."
+    )
+    private var licenseId: String? = null
+
     override fun runCommand(jc: JCommander): Int {
         val ortResult = ortResultFile.expandTilde().readValue<OrtResult>()
         val copyrightGarbage = copyrightGarbageFile?.expandTilde()?.readValue<CopyrightGarbage>().orEmpty()
 
         val copyrightStatements = ortResult
             .getProcessedCopyrightStatements(copyrightGarbage = copyrightGarbage.items)
+            .filter { (id, _) -> packageId == null || id == packageId }
```

Comment:
Nit: Could be `.filterNot { (id, _) -> id != packageId }`, similar below.

Reply:
Done.